src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/MessageProcessor.cs

```diff
@@ -71,8 +71,6 @@ public virtual Task CompleteProcessingMessageAsync(Message message, FunctionResu
                 throw new ArgumentNullException(nameof(result));
             }
 
-            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
we have the same pattern in `SessionMessageProcessor`, do we want to change it also?

Reply:
Good catch!   Updated that class as well.

Reply:
FYI, I have removed this logic as it impacts drain mode by abandoning all in-flight messages (execution started before but did not complete by the time StopAsync() is called) which defeats the purpose of the draining logic.